posthog/user_scripts/latest_user_defined_function.xml

```diff
@@ -442,10 +442,6 @@ This file is autogenerated by udf_versioner.py. Do not edit this, only edit the
             <type>UInt8</type>
             <name>num_steps</name>
         </argument>
-        <argument>
-            <type>UInt8</type>
-            <name>num_steps</name>
-        </argument>
```

Comment:
why is that removed?

Reply:
huh that's a good catch. i actually don't know since I just ran the script and didn't touch these parts on the original file. i'll see what went wrong on the generation

Reply:
it's a bug, safe to be removed. confirmed with @aspicer